src/components/taskDetails/index.tsx

```diff
@@ -64,6 +64,8 @@ type Props = {
 
 const TaskDetails: FC<Props> = ({ taskID }) => {
     const router = useRouter();
+    const { dev } = router.query;
+    const isDev = dev === 'true';
```

Comment:
```suggestion
    const { dev } = router.query;
    const isDev = dev === 'true';
```
Can we do it like this?
```
const isDev = router.query.dev === "true";
```
because dev parameter is only being used in line 67 and 68.

Reply:
resolved